### ISSUE TITLE
[Global] [Native] Update Jetpack Compose code samples to support sufficient color contrast in light and dark modes

### ISSUE THREAD

**Issue by rnguyen253** [tmobile/magentaA11y#388]
Currently, the Jetpack Compose examples in various Native App criteria pages does not have sufficient color contrast in dark mode. 

Criteria where this is an issue include: 

Alert Dialog (see attached screenshot)

Modal

Snackbar / Toast

Toggle Switch

Time Picker

Slider

Sheet

Segmented Controls

Menu

Link

Dropdown

Date Time Picker

Chip

Checkbox

Carousel

Calendar Date Picker

**Comment by danbivins** [comment]
Closed since the PR above was approved and merged.
